plugins/contextmenu/plugin.js

```diff
@@ -163,7 +170,7 @@ CKEDITOR.plugins.add( 'contextmenu', {
 		var contextMenu = editor.contextMenu = new CKEDITOR.plugins.contextMenu( editor );
 
 		editor.on( 'contentDom', function() {
-			contextMenu.addTarget( editor.editable(), editor.config.browserContextMenuOnCtrl !== false );
+			contextMenu.addTarget( editor.editable(), editor.config.contextMenuSwitchingEnabled !== false ,editor.config.nativeContextMenuOnByDefault == false );
```

Comment:
This is a breaking change as `browserContextMenuOnCtrl` is not supported anymore. IN CKEditor 4 we try to keep backwards compatibility between versions so such change should introduce new config options along with the old ones (and not changing how they worked before).